### PR TITLE
Bind command in code-behind instead of XAML. Fixes #361

### LINKED ISSUE #361: Command Binding Example /docs/guidelines/framework/commands
Currently example looks like:
```
// In XAML
<Button Command="{Binding Delete}" .../>public class RepositoryViewModel : ReactiveObject
{
 public RepositoryViewModel()
 {
   Delete = ReactiveCommand.CreateFromObservable(DeleteImpl);
   Delete.IsExecuting.ToProperty(this, x => x.IsDeleting, out _isDeleting);
   Delete.ThrownExceptions.Subscribe(ex => this.Log().ErrorException("Something went wrong", ex));
 }  public ReactiveCommand<Unit, Unit> Delete { get; private set; }  readonly ObservableAsPropertyHelper<bool> _isDeleting;
 public bool IsDeleting { get { return _isDeleting.Value; } }  public IObservable<Unit> DeleteImpl()
 {
   return Observable.Start(() => /* ... */);
 }
}
```
see: https://reactiveui.net/docs/guidelines/framework/commands

Command should be in Codebehind like:

`this.BindCommand(ViewModel, vm => Delete, v => v.DeleteButton);`

### PR DESCRIPTION
**Pull request by Lixfeld**
**What are the main goals of this change?**
- [ ] Better readability (style, grammar, spelling...)
- [x] Content correction (accuracy, wrong information...)
- [x] Remove duplicates

### CODE REVIEW

File: input/docs/handbook/logging/debugging-observables.md [hunk 1]
```diff
@@ -1,22 +0,0 @@
-### Debugging Observables
```
Comment (glennawatson) [inline input/docs/handbook/logging/debugging-observables.md]: Sorry for the delay in reviewing this one. Just curious why this file was deleted?

Comment (Lixfeld) [reply input/docs/handbook/logging/debugging-observables.md]: It's a duplicate (see: https://reactiveui.net/docs/handbook/logging/#debugging-observables)
but I could also be replaced with a link to the updated page.

File: input/docs/handbook/commands/index.md [hunk 2]
```diff
@@ -76,7 +74,7 @@ LoadUsers = ReactiveCommand.CreateFromTask(LoadUsersAsync);
 // LoadUsers Observable, update our OAPH and notify the UI 
 // that the value of Users property has changed.
 _users = LoadUsers.ToProperty(
-    this, x => x.Users, RxApp.MainThreadScheduler
+    this, x => x.Users, scheduler: RxApp.MainThreadScheduler);
```
Comment (worldbeater) [inline input/docs/handbook/commands/index.md]: Looks like the syntax became invalid, now we have two `);` here.

File: input/docs/handbook/commands/index.md [hunk 3]
```diff
@@ -15,9 +15,8 @@ A `ReactiveCommand` is created using static factory methods which allows you to
 // A synchronous command taking a parameter and returning nothing.
 // The Unit type is often used to denote the successfull completion
 // of a void-returning method (C#) or a sub procedure (VB).
-var command = ReactiveCommand.Create<int, Unit>(
-    integer => Console.WriteLine(integer)
-);
+ReactiveCommand<int,Unit> command = ReactiveCommand.Create<int>(
```
Comment (worldbeater) [inline input/docs/handbook/commands/index.md]: Which ReactiveUI version you are using? Afair, the syntax `.Create<TIn>` was deprecated a long time ago, and now one needs to specify both `TIn` and `TOut` types explicitly.

File: input/docs/handbook/commands/index.md (hunk 2 above)
Comment (Lixfeld) [reply input/docs/handbook/commands/index.md]: Thank you for pointing out my mistake.

File: input/docs/handbook/commands/index.md (hunk 3 above)
Comment (Lixfeld) [reply input/docs/handbook/commands/index.md]: I'm using the latest version (11.1.6) and I get the following error:
`Cannot implicitly convert type 'void' to 'System.Reactive.Unit'`
Removing `Unit` in `TOut` fixes the error and I changed `var` to the whole command to make that clear. Is there another way to fix this error?

Comment (worldbeater) [reply input/docs/handbook/commands/index.md]: Hmm, thank you for pointing this out and submitting a fix! Looks like I missed that we actually have the [`Create<TParam, TResult>`](https://github.com/reactiveui/ReactiveUI/blob/master/src/ReactiveUI/ReactiveCommand/ReactiveCommand.cs#L220), [`Create<TParam>`](https://github.com/reactiveui/ReactiveUI/blob/master/src/ReactiveUI/ReactiveCommand/ReactiveCommand.cs#L175) and [`Create<TResult>`](https://github.com/reactiveui/ReactiveUI/blob/master/src/ReactiveUI/ReactiveCommand/ReactiveCommand.cs#L135) factory method overloads now, this seems to have been changed in one of the latest ReactiveUI releases. The `api/reactiveui/reactivecommand` stuff gets generated from [ReactiveCommand API docs](https://github.com/reactiveui/ReactiveUI/blob/master/src/ReactiveUI/ReactiveCommand/ReactiveCommand.cs#L42) btw. Thanks again for submitting the fix!

Comment (glennawatson) [reply input/docs/handbook/commands/index.md]: Been that way for a while @worldbeater. We deprecated the non generic ReactiveCommand class a while back.

**Pull request merged**